docs/source/Support/bskReleaseNotes.rst

```diff
@@ -41,6 +41,9 @@ Version 2.4.0 (August 23, 2024)
 - Added a new example scenario :ref:`scenarioConstrainedDynamics` demonstrating post-docked spacecraft dynamics
 - Created a :ref:`constraintDynamicEffector` dynamics module to couple separate spacecraft motion using holonomic
   constraints
+- Created an input device status message to toggle the :ref:`constraintDynamicEffector` dynamics module on/off.
```

Comment:
This needs to be moved to the current BSK release cycle release notes.

Reply:
Sorry for the oversight. This has now been updated